server.js

```diff
@@ -1,9 +1,9 @@
 import express from 'express';
-import router from './server/routes';
+import router from './server/routes/index';
```

Comment:
If the file is named `index.js` you don't need to include it here. JS automatically looks for `index.js` if the `path to import` is a folder.